src/main/xerus/monstercat/tabs/TabCatalog.kt

```diff
@@ -13,19 +14,21 @@ import xerus.ktutil.javafx.ui.controls.*
 import xerus.ktutil.toLocalDate
 import xerus.monstercat.Settings
 import xerus.monstercat.api.Player
+import xerus.monstercat.api.Playlist
+import xerus.monstercat.api.Song
 import xerus.monstercat.logger
 import java.time.LocalTime
 import java.util.*
 import kotlin.math.absoluteValue
 
 class TabCatalog : TableTab() {
-	
```

Comment:
fix this in your codestyle please

Reply:
There should be a return here ?

Reply:
Nah look at the intendation throughout this file

Reply:
Finally fixed